deploy/chart/values.yaml

```
@@ -101,3 +101,6 @@ images:
   pluginRegistrar:
     repo: acs/csi-node-driver-registrar
     tag: "v2.9.0-d48d2e0-aliyun"
+  ossfs:
+    tag: "v1.88.4-d9f3917-aliyun"
+    updatedTag: "v1.91.3-d9f3917-aliyun"
```

```suggestion
  ossfs:
    repo: acs/ossfs
    tag: "v1.88.4-d9f3917-aliyun"
  ossfsUpdated:
    repo: acs/ossfs
    tag: "v1.91.3-d9f3917-aliyun"
```

Please use consistent format. I'm using this to verify image sync before releasing.